GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockIII.cfg

```diff
@@ -76,18 +102,19 @@ PART
 	}
 }
 
+//	================================================================================
+//	Configure RealAntennas
+//	================================================================================
+// A steerable unified S-band high-gain antenna for long-range communications with Earth was mounted on the aft bulkhead.
+// https://en.wikipedia.org/wiki/Apollo_command_and_service_module#Communications_system
+
 @PART[ROC-ApolloHGABlockIII]:BEFORE[RealAntennas]:NEEDS[RealAntennas]
 {
 	!MODULE[ModuleDataTransmitter],* {}
 	MODULE
 	{
 		name = ModuleRealAntenna
 		antennaDiameter = 0.9	//Guess, no documentation
+		RFBand = S	//Default to S-band
 	}
-	MODULE
-	{
-		name = ModuleRealAntenna
-		referenceGain = 3.0		//Guess based on Block II
-		RFBand = S
-	}
-}
\ No newline at end of file
+}
```

Comment:
The ModuleRealAntenna modules were intentionally duplicated to provide both an omni and a steerable dish.  Is that not correct?